<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/prefix1/icon/app_icon.ico</normaloff>:/new/prefix1/icon/app_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnExec">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="proBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>711</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>721</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>1-请选择路径</string>
    </property>
    <widget class="QLineEdit" name="lEdit2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lEdit1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输出目录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入目录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnSel2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>选择...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnSel1">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>选择...</string>
     </property>
    </widget>
    <zorder>lEdit1</zorder>
    <zorder>label_2</zorder>
    <zorder>label</zorder>
    <zorder>tbtnSel2</zorder>
    <zorder>tbtnSel1</zorder>
    <zorder>lEdit2</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>721</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>2-请输入经纬度范围值</string>
    </property>
    <widget class="QLineEdit" name="editMaxLon">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMaxLat">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMinLon">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMinLat">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>80</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>最大经度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>最小经度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>最大纬度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>最小纬度</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>3-请选择功能</string>
    </property>
    <widget class="QRadioButton" name="rbCopy">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;复制&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>复制</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbPaste">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>剪切</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbDelete">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>691</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShow">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查看报告</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnList">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>收起报告</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
